website/docs/guides/markdown-features/markdown-features-admonitions.mdx

```diff
@@ -11,61 +11,61 @@ Example:
 
     :::note
 
-    The content and title *can* include markdown.
+    Some **content** with _markdown_ `syntax`.
```

Comment:
There's not much value to have a custom message for each admonition type.
We'd rather cover all the edge cases for ALL types so that issues are easily surfaced in the docs. In fact Im going to add a link with a nested code block to all items to cover more cases (that are already covered in Infima demo btw)